frontend/src/css/password.css

```diff
@@ -0,0 +1,83 @@
+#logo {
+  float: left;
+  padding-top: 20px;
+  padding-left: 20px;
+}
+
+#text {
+  color: #054D74;
+  width: 575px;
+  height: 265px;
+  /* background-color: brown; */
+  
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  
+  margin: auto;
+  text-align: left;
+}
+
+#forgotPassword {
+  margin-top: 0px;
+  float: left;
+  font-size: 57.7px;
```

Comment:
For the CSS of the forgot password page can we make the font smaller and be in the center of the screen? Same with the enterEmailText and the input field. I think using [Material-UI text fields](https://mui.com/components/text-fields/) would make the inputs look better for all pages as well. Also if we can make it less wide that would be great.